util/additional-measures.json

```diff
@@ -80,5 +80,33 @@
     "measureSets": [],
     "isBonus": false,
     "objective": "electronicPrescribing"
-  }
+  },
+  {
+    "category": "aci",
+    "firstPerformanceYear": 2017,
+    "lastPerformanceYear": null,
+    "metricType": "boolean",
+    "measureId": "ACI_LVSSC_1",
+    "title": "Low-Volume Send a Summary of Care",
```

Comment:
These came from the text in the ticket, where "Send a Summary of Care" and "Request / Accept a Summary of Care" appear to be "titles" of the original measures: [QPPA-419](https://jira.cms.gov/browse/QPPA-419)

But I like your titles better still so updated: https://github.com/CMSgov/qpp-measures-data/pull/18/commits/9a4908daed544e55e8c05c715fd0c5b1ecca2682